src/SapiEmitter.php

```diff
@@ -98,6 +98,13 @@ public function emit(ResponseInterface $response, bool $withoutBody = false): vo
             }
         }
 
+        /**
+         * Sends headers before the body.
+         * Makes a client possible to recognize the type of the body content if it is sent with a delay,
+         * for instance, for a streamed response.
+         */
+        flush();
```

Comment:
Add tests for this. Seems it can be implemented via `ob_start()`: `flush` will be call before `ob_end_flush`.